youtube_dl/extractor/generic.py

```diff
@@ -3119,9 +3119,13 @@ def _real_extract(self, url):
         jwplayer_data = self._find_jwplayer_data(
             webpage, video_id, transform_source=js_to_json)
         if jwplayer_data:
-            info = self._parse_jwplayer_data(
-                jwplayer_data, video_id, require_title=False, base_url=url)
-            return merge_dicts(info, info_dict)
+            try:
+                info = self._parse_jwplayer_data(
+                    jwplayer_data, video_id, require_title=False, base_url=url)
+                return merge_dicts(info, info_dict)
+            except ExtractorError, e:
+                self.to_screen(e.msg_without_bug_report)
+                self.to_screen("Trying different extractor")
```

Comment:
No.

Reply:
I'm not sure what's wrong with this message. Should I just remove them.

Reply:
Feedback would be appreciated.

Reply:
There should not be any messages.